puppet/puppet-manifests/personalize-desktop.pp

```diff
@@ -0,0 +1,35 @@
+#node default { }
+
+node 'dev-box' {
+
+	exec { 'make dev user default':
+		command => '/bin/sed -i "s/vagrant/dev/g" /etc/lightdm/lightdm.conf',
+	}
+	
+	exec { 'add osgp wallpaper':
+		command => '/bin/cp -f /home/vagrant/repos/Config/vagrant/OSGP-solar.png /usr/share/backgrounds/warty-final-ubuntu.png',
```

Comment:
@smvdheijden 
This is exactly what I tried to do first. This worked perfectly for the Vagrant user. But it didnt work for the dev user. For some reason this only seems to work when a user has already logged in at least once. This is unfortunately not true for the dev user..